documentation/manpages/sdk/dotnet-pack.1

```diff
@@ -164,6 +164,13 @@ Doesn\[cq]t display the startup banner or the copyright message.
 .RS 2
 .PP
 Places the built packages in the directory specified.
+.IP \[bu] 2
+\&.NET 7.0.200 SDK
+.RS 2
+.PP
+In the 7.0.200 SDK, if you specify the \f[V]--output\f[R] option when running this command on a solution, the CLI will emit an error.
```

Comment:
does this error in 7.0.100 as well? 7.0.200 shipped after .1xx

Reply:
We actually removed this warning entirely for pack in 201 - the source documentation needs updating.

Reply:
lol, ignore me - it actually says this right below.

to answer your question @joeloff it does not error in 7.0.100 - this was an inadvertent breaking change that slipped into 7.0.200 that we rightfully got lambasted for.